pkg/util/os/os.go

```diff
@@ -43,3 +47,36 @@ func CurrentOS() OS {
 	}
 	panic("Unexpected OS type")
 }
+
+func CurrentExecutable() (string, error) {
+	currentExec, err := osext.Executable()
+	if err != nil {
+		return "", err
+	}
+	return currentExec, nil
+}
+
+// EnvironmentAsMap returns os.Environ as map.
+func EnvironmentAsMap() map[string]string {
```

Comment:
Why, then I would need to pull out 'os.Environ()' as well. I think it is a valid helper method for dealing with the current environment.